docs/how-to-use.html

```diff
@@ -9,7 +9,7 @@
     <meta name="googlebot" content="index,follow,snippet,archive">
 
     <title>TruLens: Explainability for LLM Apps</title>
-    <meta name="author" content="Maros Janota">
```

Comment:
Note: Fairly certain this is a remnant from a template. Updating to "TruEra" as this person is neither affiliated with TruEra or TruLens.